EIPS/eip-2488.md

```diff
@@ -0,0 +1,58 @@
+---
+eip: 2488
+title: Deprecate the CALLCODE opcode
+author: Alex Beregszaszi (@axic)
+discussions-to: https://ethereum-magicians.org/t/eip-2488-deprecate-the-callcode-opcode/3957
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-12-20
+requires: 7
+---
+
+## Abstract
+
+Deprecate `CALLCODE` in a *somewhat* backwards compatible way, by making it always return failure.
```

Comment:
```suggestion
Deprecate `CALLCODE` (`0xf2`) in a *somewhat* backwards compatible way, by making it always return failure.
```

Reply:
I think this (and other comments) aligns with the new OPCODE referencing systeme that you recently proposed.

Reply:
Hmm, after leaving all of these suggestion comments, maybe the intent is that this style is *only* applied in the `Specification` section?  If so, feel free to reject all of the suggestions that just add the parenthetical callcode.

Reply:
EIP-1 has this (from #1968):

![Screenshot](https://user-images.githubusercontent.com/20340/91622445-0c0d1300-e98f-11ea-81cb-d356446df988.png)

So it is not super clear. I would be inclined to say mentioning it at the first reference in the Specification is a reasonable option, otherwise it would become annoying.